sdk/test/resource/resource_test.cc

```diff
@@ -173,13 +174,21 @@ TEST(ResourceTest, OtelResourceDetector)
     }
   }
   EXPECT_EQ(received_attributes.size(), expected_attributes.size());
+#if defined(_WIN32)
+  putenv("OTEL_RESOURCE_ATTRIBUTES=");
+#else
   unsetenv("OTEL_RESOURCE_ATTRIBUTES");
```

Comment:
There is generally a huge issue with the env var approach for Win 10 Universal apps, i.e. someone ever trying to compile this for Xbox is gonna be having some fun time with environment-variables based approach. Not a blocker for this PR, but something we need to revisit in the `putenv` / resource API :

https://github.com/MicrosoftDocs/cpp-docs/blob/master/docs/cppcx/crt-functions-not-supported-in-universal-windows-platform-apps.md

Reply:
Thanks for calling it out. Yes, let's revisit this later.